users/miles2go/keymaps/handwired/ms_sculpt_mobile/keymap.c

```diff
@@ -0,0 +1,462 @@
+#include QMK_KEYBOARD_H
+
+
+#include "virtser.h"
+#include <print.h>
+#include "milestogo.h"
+
+
+#define LAYOUT_local LAYOUT_mobile_XUW
+#define LAYOUT LAYOUT_mobile_XUW
+
+
+#ifndef USERSPACE_ACTIVE
+enum layer_keycodes {
+    QWR,
+    CDH,
+    SYM,
+    MOV,
+    NUM,
+    TRAN
+};
+
+enum layer_names {
+_QWR =0,
```

Comment:
You don't need the 0 

```suggestion
_QWR,
```